client/my-sites/stats/post-performance/index.jsx

```diff
@@ -189,9 +129,12 @@ export default connect( ( state, ownProps ) => {
 	const query = { status: 'published' };
 	const posts = site ? getSitePostsForQuery( state, site.ID, query ) : null;
 	const post = posts && posts.length ? posts[ 0 ] : null;
+	const countViews = post && site ? getPostStat( state, 'views', site.ID, post.ID ) : null;
+
 	return {
+		countViews,
 		query,
 		post,
-		loading: isRequestingSitePostsForQuery( state, site.ID, query )
+		loading: !site || !posts || countViews === null
```

Comment:
Small formatting changes here, spaces needed between the `!` and the variable names ( `! site` ).